libs/features/mas/src/global.css.js

```diff
@@ -247,6 +248,45 @@ merch-card.has-divider div[slot='body-lower'] hr {
     margin: 0;
 }
 
+/* HR-based dividers from Studio */
+merch-card .divider-wrapper {
+    display: block;
+    margin: 8px 0;
+}
+
+merch-card .divider-hr {
+    border: none;
+    margin: 0;
+    background-color: var(--consonant-merch-card-border-color, #e9e9e9);
+}
+
+merch-card .divider-hr.divider-size-s {
```

Comment:
we only have one type of divider according to figma, could you pls remove 'size-s' etc.